deployment/live/aws/conformance/terragrunt.hcl

```diff
@@ -14,8 +14,8 @@ locals {
   signer                     = get_env("TESSERA_SIGNER")
   verifier                   = get_env("TESSERA_VERIFIER")
   # Roles are defined externally
-  ecs_execution_role        = "arn:aws:iam::864981736166:role/ecsTaskExecutionRole"
-  ecs_conformance_task_role = "arn:aws:iam::864981736166:role/ConformanceECSTaskRolePolicy"
+  ecs_execution_role        = "arn:aws:iam::703671898905:role/ecsTaskExecutionRole"
```

Comment:
When you're done testing, these need to be modified again

Reply:
My plan was to remove this commit before merging. 👍